feat(django-homework-assignment2): Add inventory status view and rela…

morristseng opened the pull request:
…ted test cases.

[why]
As for assignment2 requirement (實作另一個 List View 頁面，可觀看目前所有的 Shipping / Receiving 加總的結果)

[how]
Using QuerySet's aggregate to get total quantity.
Also wrote 3 tests to cover both simple and complex cases.

morristseng committed:
feat(django-homework-assignment2): Add inventory status view and related test cases.

[why]
As for assignment2 requirement (實作另一個 List View 頁面，可觀看目前所有的 Shipping / Receiving 加總的結果)

[how]
Using QuerySet's aggregate to get total quantity.
Also wrote 3 tests to cover both simple and complex cases.

alfredweihc commented on inventory/views.py:
```diff
@@ -1,27 +1,52 @@
+    page_num = int(request.GET.get('page', '1'))
+    page_size = int(request.GET.get('o', '50'))
+    # Question:
+    # Why not just use pagination from django rest framework?
```
可以啊，作業應該也是用 django paginator

alfredweihc commented on inventory/tests.py:
```diff
@@ -1,3 +1,46 @@
+
+
+def create_basic():
+    parner = TradeParner.objects.create(name='parner1', address='add1')
```
nit: baker sometimes quicker

alfredweihc reviewed (approved):
nit: some files don't follow PEP8
logic part overall LGTM

morristseng committed:
feat(django-homework-assignment2): Apply pep8 style, using baker for tests.

[why]
Addressed reviewer's comment

[how]
1. Apply pep8 coding convention.
2. Using baker for tests.
